tutorials/dataframe/df016_vecOps.py

```diff
@@ -15,7 +15,8 @@
 import ROOT
 
 df = ROOT.RDataFrame(1024)
-coordDefineCode = '''ROOT::VecOps::RVec<double> {0}(len);
+coordDefineCode = '''using namespace ROOT;
+                     RVecD {0}(len);
```

Comment:
```suggestion
coordDefineCode = '''ROOT::RVecD {0}(len);
```

Reply:
This and the other remark were resolved.